### PR TITLE
fix: should color style  works on Badge

### PR DESCRIPTION
**Pull request by DiamondYuan**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. Describe the source of requirement, like related issue link.

2. Describe the problem and the scenario.

### 💡 Solution

1. How to fix the problem, and list final API implementation and usage sample if that is an new feature.

2. GIF or snapshot should be provided if includes UI/interactive modification.

### 📝 Changelog description

> Describe changes from userside, and list all potential break changes or other risks.

1. English description

2. Chinese description (optional)

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by DiamondYuan** [comment]
https://github.com/ant-design/ant-design/issues/15349

**Comment by DiamondYuan** [comment]
● renders ./components/tabs/demo/custom-tab-bar-node.md correctly

挂了？

**Comment by DiamondYuan** [comment]
@zombieJ  ci broken

File: components/badge/demo/status.md [hunk 1]
```diff
@@ -33,6 +37,8 @@ ReactDOM.render(
     <Badge status="processing" text="Processing" />
     <br />
     <Badge status="warning" text="Warning" />
+    <br />
+    <Badge style={style} status="success" text="Success" />
```
Comment (zombieJ) [inline components/badge/demo/status.md]: 不用特地放在 demo 里。

File: components/badge/__tests__/index.test.js [hunk 2]
```diff
@@ -89,4 +89,10 @@ describe('Badge', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  // https://github.com/ant-design/ant-design/issues/15349
+  it('should color style  works on Badge', () => {
+    const wrapper = render(<Badge style={{ color: 'red' }} status="success" text="Success" />);
+    expect(wrapper.find('.ant-badge-status-text').get(0).attribs.style).toBe('color:red');
```
Comment (zombieJ) [inline components/badge/__tests__/index.test.js]: => `xxx.props().style.color` & `toBe('red')`

**Comment by zombieJ** [comment]
> @zombieJ ci broken

merge master

**Comment by DiamondYuan** [comment]
> > @zombieJ ci broken
> 
> merge master

master 跪了。

**Comment by zombieJ** [comment]
嗯，enzyme bug：https://github.com/airbnb/enzyme/issues/2046

**Pull request merged**